api/tests/unit/domain/usecases/find-certification-assessments_test.js

```diff
@@ -1,19 +1,18 @@
 const { expect, domainBuilder, sinon } = require('../../../test-helper');
+const assessmentRepository = require('../../../../lib/infrastructure/repositories/assessment-repository');
 const findCertificationAssessments = require('../../../../lib/domain/usecases/find-certification-assessments');
 
 describe('Unit | UseCase | find-certification-assessments', () => {
 
-  const assessmentRepository = {
-    getCertificationAssessmentByUserIdAndCourseId: () => {
-    },
-  };
+  beforeEach(() => {
+    sinon.stub(assessmentRepository, 'getCertificationAssessmentByUserIdAndCourseId');
+  });
 
-  it('should resolve an empty array', () => {
+  it('should resolve an empty array since courseId in null', () => {
```

Comment:
typo `in`/ `is`

Peut-être faudrait-il dire "when courseId is null" plutôt que `since`?